openupgradelib/openupgrade.py

```diff
@@ -632,9 +632,16 @@ def rename_columns(cr, column_spec):
 
 
 def rename_fields(env, field_spec, no_deep=False):
```

Comment:
Would it make sense to allow an `env_or_cr` attribute here and deprecate passing an env when the major Odoo version is 17 or higher?

Reply:
Suppose someday a PR like https://github.com/OCA/openupgradelib/pull/298 gets merged.

In this case, we would have:

```python
def rename_fields(env, field_spec, no_deep=False):
    rename_fields_cr(env.cr, field_spec)
    if no_deep:
        _rename_field_on_custom_view(env, ...)

def rename_fields_cr(cr, field_spec):
    ...
```

If we go by `env_or_cr`, and forcing cr in a version, then this version will not have this extra method that uses env.

Reply:
ok